code/datums/supplyexports.dm

```diff
@@ -10,7 +10,7 @@
 
 
 /datum/supply_export/xemomorph/larva
-	cost = 0
+	cost = 3
```

Comment:
Undocumented change. Also undesirable for larvas to cost more than a single point, really. We'd have to check for embryos as well.